pkg/adapter/main.go

```diff
@@ -36,6 +36,7 @@ import (
 	"knative.dev/pkg/profiling"
 	"knative.dev/pkg/signals"
 	"knative.dev/pkg/source"
+
```

Comment:
nit: this new line looks like a oops

Reply:
interesting enough, it was placed by the Goland formatter :thinking: